packages/core/src/components/templates/SessionListCard/SessionListCard.js

```diff
@@ -0,0 +1,363 @@
+        />
+      </View>
+      <ScrollView
+        contentContainerStyle={styles.contentContainer}
+        style={styles.scrollView}>
+        {sessions.map((_session, index) => {
+          return (
+            <View key={index} style={styles.listContainer}>
+              <TouchableOpacity
+                onLongPress={() => {
+                  changeActiveSession(_session.sessionId);
+                  onSwipeLeft();
+                }}
+                onPress={() => {
+                  onSwipeRight();
+                  changeActiveSession(_session.sessionId);
+                }}
+                activeOpacity={0.9}>
+                <Animated.View
+                  onTouchStart={onTouchStart}
+                  onTouchEnd={onTouchEnd}
+                  style={[
+                    selectedIndex === index
+                      ? {
+                          transform: [
+                            {translateX: translateXAnim},
+                            {translateY: translateYAnim},
+                            {scale: scaleAnim},
+                          ],
+                        }
+                      : null,
+                  ]}>
+                  <Card
+                    style={[
+                      styles.cardContainer,
+                      _session?.sessionId === session?.sessionId
+                        ? styles.selectBorderCard
+                        : styles.borderCard,
+                      _session?.isDefault ? styles.borderIsDefaultCard : null,
+                    ]}>
+                    <View style={styles.imageContainer}>
+                      <LogoImage logoFile={logoFile} url={_session?.url} />
+                    </View>
+                    <View style={styles.textContainer}>
+                      <Text style={styles.textTitle}>{_session.id}</Text>
+                      <Text
+                        numberOfLines={1}
+                        style={styles.details}>{`Url: ${_session.url}`}</Text>
+                    </View>
+                  </Card>
+                </Animated.View>
+              </TouchableOpacity>
+              {showButton && _session?.sessionId === session?.sessionId && (
+                <Animated.View
+                  style={[
+                    styles.squareIconContainer,
+                    {transform: [{translateX: translateXAnim}]},
+                  ]}>
+                  <Icon
+                    name="pencil-alt"
+                    style={styles.iconPen}
+                    touchable={true}
+                    onPress={() => setPopupEditIsOpen(true)}
+                  />
+                  <Icon
+                    name="trash-alt"
+                    style={styles.iconTrash}
+                    touchable={true}
+                    onPress={() => animateRemoval(index)}
+                    color={Colors.errorColor.background}
+                  />
+                </Animated.View>
+              )}
+            </View>
```

Comment:
Should be done in another component as discussed
Also, issue with icon display on swipe, please check what's wrong
Do not display blue border please